### PR TITLE
BCDA-4239 - Add method to generate ALR jobs in service.go

### PR DESCRIPTION
**Pull request by mtrang1263**
### Fixes [BCDA-4239](https://jira.cms.gov/browse/BCDA-4239)

Adding method to service package to generate ALR jobs.

In this PR, we are piloting the usage of [testfixtures](https://github.com/go-testfixtures/testfixtures). These fixtures allow us to load data into our test postgres database via fixtures file. This will allow our test code to focus on validation instead of dealing with the loading of data.

### Change Details
1. Added alr.go to the service package that is responsible for creating ALR jobs.
2. Added new utilities to create test databases while ensuring consistent schemas. This will allow us to leverage the testfixtures approach while still allowing for parallel tests.

### Security Implications
- [x] new software dependencies
We are using two new test scoped dependencies.

1. https://github.com/go-testfixtures/testfixtures - Used for loading the database via files
2. https://github.com/golang-migrate/migrate - Used to ensure consistent schema in the test database.

- [ ] security controls or supporting software altered
- [ ] new data stored or transmitted
- [ ] security checklist is completed for this change
- [ ] requires more information or team discussion to evaluate security implications
- [x] no PHI/PII is affected by this change

### Acceptance Validation
CI Passes. This functionality is not exposed to the caller. We have follow up tickets to achieve that.
### Feedback Requested
Review testing strategy to see if it will meet the needs of our improved testing effort.

### CODE REVIEW

File: bcda/service/alr_test.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+			)
+			for {
+				part, start = partitionBenes(start, tt.size)
+				if len(part) == 0 {
+					break
+				}
+				numParts++
+				total = append(total, part...)
+			}
+			assert.Equal(t, benes, total)
```
Comment (CosmicSyntax) [inline bcda/service/alr_test.go:128]: assert.Equal(t, numParts, expNumParts)?

File: bcdaworker/queueing/enqueue.go [hunk 2]
```diff
@@ -58,3 +60,19 @@ func (q queEnqueuer) AddJob(job models.JobEnqueueArgs, priority int) error {
 
 	return q.Enqueue(j)
 }
+
+// ALR ENQ...
+func (q queEnqueuer) AddAlrJob(job models.JobAlrEnqueueArgs, priority int) error {
```
Comment (CosmicSyntax) [inline bcdaworker/queueing/enqueue.go:65]: Should we add this to enqueue_test.go?

Comment (mtrang1263) [reply bcdaworker/queueing/enqueue.go:65]: Yup! But it's going to be in your PR (https://github.com/CMSgov/bcda-app/pull/660/files#diff-6142142004eef8af8589b36ab24ec666a3f15b4e15593a2fa928038fbed7af62R169)

**Comment by mtrang1263** [comment]
Getting  "Failed to detect issues" error message.

Per [Snyk](https://support.snyk.io/hc/en-us/articles/360000920678-Failed-to-detect-issues), will recreate the PR.

**Pull request closed without merging**